### PR TITLE
Add some screenshots for ParleyMessageView

### PR DESCRIPTION
**Pull request by nlamah**
Add some screenshots for ParleyMessageView
Fix time label for ParleyMessageView when using dynamic font sizing

### CODE REVIEW

File: Package.resolved [hunk 1]
```diff
@@ -1,43 +1,59 @@
 {
-  "object": {
```
Comment (nlamah) [inline Package.resolved:2]: SwiftFormat added the commas

File: Sources/Parley/Views/ParleyMessageView/ParleyMessageView.xib [hunk 2]
```diff
@@ -237,7 +237,7 @@
                                 <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="ITI-5G-N0m" userLabel="Time view">
                                     <rect key="frame" x="0.0" y="798" width="414" height="16"/>
                                     <subviews>
-                                        <stackView opaque="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="OoS-tA-TUD">
+                                        <stackView opaque="NO" contentMode="scaleToFill" alignment="center" translatesAutoresizingMaskIntoConstraints="NO" id="OoS-tA-TUD">
```
Comment (nlamah) [inline Sources/Parley/Views/ParleyMessageView/ParleyMessageView.xib:240]: The icon height constraints was limiting the `TimeLabel`.

File: Sources/Parley/Views/ParleyMessageView/ParleyMessageView.swift [hunk 3]
```diff
@@ -195,6 +198,7 @@ final class ParleyMessageView: UIView {
         
         imageNameLabel.text = message.agent?.name
         nameLabel.text = message.agent?.name
+        nameLabel.adjustsFontForContentSizeCategory = true
```
Comment (nlamah) [inline Sources/Parley/Views/ParleyMessageView/ParleyMessageView.swift:201]: The name label was not changing fontSize when Dynamic Font Category was altered.

**Review by alexkok — APPROVED** [review]
Looks good, thanks! 
Only thing that is test-visually incorrect is that the user messages have an `Title` where this cannot happen logically in Parley -- Only agent messages have these.

Maybe we should adjust these tests such that the `Title` is gone for user screenshots?

**Comment by nlamah** [comment]
> Looks good, thanks! Only thing that is test-visually incorrect is that the user messages have an `Title` where this cannot happen logically in Parley -- Only agent messages have these.
> 
> Maybe we should adjust these tests such that the `Title` is gone for user screenshots?

I updated the screenshots.

**Comment by nlamah** [comment]
When will the pull request be merged to develop branch?

**Comment by nlamah** [comment]
@alexkok I added some more (minor) commits. Is your approval still valid after these commits?

**Comment by alexkok** [comment]
Perfect! Thanks for the headsup @nlamah, reviewed again and looks good 👍

**Pull request merged**